test/integration/standup.test.js

```diff
@@ -0,0 +1,197 @@
+          expect(userData.roles).to.have.property("monitored");
+          expect(userData.roles.monitored).to.be.equal(true);
+          return done();
+        });
+    });
+    it("Checks superuser can un-mark a user from monitoring", function (done) {
+      chai
+        .request(app)
+        .post(`/standup/${userId}`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`)
+        .send({
+          monitor: false,
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          const userData = userQuery.fetchUser({ userId: userId });
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a(object);
+          expect(res.body.message).to.equal("User marked for standup successfully.");
+          expect(userData.roles).to.have.property("monitored");
+          expect(userData.roles.monitored).to.be.equal(false);
+          return done();
+        });
+    });
+    it("Should return 401 for other/un-authenticated users", function (done) {
+      chai
+        .request(app)
+        .post(`/standup/${userId}`)
+        .set("Cookie", `${cookieName}=${jwt}`)
+        .send({
+          monitor: true,
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(401);
+          expect(res.body).to.be.a(object);
+          expect(res.body.message).to.equal("Unauthenticated User");
+          return done();
+        });
+    });
+    it("Should return 404 for invalid user id", function (done) {
+      chai
+        .request(app)
+        .post(`/standup/123`)
+        .set("Cookie", `${cookieName}=${jwt}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(404);
+          expect(res.body).to.be.a(object);
+          return done();
+        });
+    });
+  });
+
+  describe("Test the get standup Api", function () {
+    it("should return the standup of specified user", function (done) {
+      chai
+        .request(app)
+        .get(`/standup/${userId}`)
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(200);
+          expect(res.body).to.be.a(object);
+          expect(res.body).to.have.property("yesterday");
+          expect(res.body).to.have.property("today");
+          expect(res.body).to.have.property("blockers");
+          expect(res.body).to.have.property("timestamp");
+          return done();
+        });
+    });
+
+    it("should return 404 for user not found", function (done) {
+      chai
+        .request(app)
+        .get("/standup/123")
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(404);
+          expect(res.body).to.be.a(object);
```

Comment:
[Nitpick]
its good to have the `expect` statement for `message` as well similar to other `it` blocks.

Reply:
added